src/markdown-pages/build-apps/permission-manage-apps.mdx

```diff
@@ -13,41 +13,124 @@ tags:
+
+### Basic users
+
+Basic users in the original user model are limited in their Nerdpack capabilities. They can only create and serve Nerdpacks locally. To publish, tag, subscribe to, and use their Nerdpacks, a basic user must be upgraded to a full user. An admin user on their account can do this for them.
+
+<Callout variant="tip">
+
+While, in general, a basic user can't use Nerdpacks, even ones that their accounts create and publish, there are some Nerdlets that New Relic maintains that basic users can use. However, these are rare and there is currently no way to distinguish them in our web interface. And even if a basic user is allowed to use a Nerdlet in a Nerdpack, they are never able to use custom visualizations. To use custom visualizations, a user must be at least a full user.
+
+</Callout>
+
+### Full users
+
+Full users in the original user model fit into two categories:
+
+- Those who have been granted the **Nerdpack Manager** add-on role
+- Those who have not been granted the **Nerdpack Manager** add-on role
+
+The **Nerdpack Manager** role is required to publish, subscribe to, and tag Nerdpacks. So, a full user who doesn't have the **Nerdpack Manager** role is only allowed to create and serve Nerdpacks locally and use Nerdpacks that their accounts have already been subscribed to.
+
+To publish, tag, or subscribe to a Nerdpack, a full user must be either upgraded to an admin user or granted the **Nerdpack Manager** add-on role by an admin.
+
+A full user with the **Nerdpack Manager** role can perform any of the Nerdpack capabilities and is, therefore, equivalent to an owner or admin within the context of Nerdpacks.
+
+### Owner/Admin users
+
+An owner or admin user can perform any of the Nerdpack capabilities, including creating, serving, publishing, tagging, subscribing to, and using any and all Nerdpack artifacts, both Nerdlets and visualizations, either built by New Relic or by their own accounts.
+
+## Nerdpack permissions: New Relic One user model
+
+On both the [original user model](https://docs.newrelic.com/docs/accounts/original-accounts-billing/original-users-roles/users-roles-original-user-model/) and the [New Relic One user model](https://docs.newrelic.com/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model-understand-user-structure/), user permissions differ between types and roles. In this section, you learn the differences between how basic users and full users can use Nerdpacks in the New Relic One user model.
```

Comment:
It would be great to avoid the sort of clunky "In this section, you learn..." sentence. Even something simple like, "Read on for information about the differences between full and basic user capabilities..." You can find a nice way to provide this info.